### ISSUE TITLE
Wrong alignment of declarations?

### ISSUE THREAD

**Issue by Koch013** [SAP/abap-cleaner#186]
Hi,

for me the alignment of chained declarations using the default action for chains (align name, type, length, value, ...) looks not as expected.

```
METHOD b.
DATA: gt_mat TYPE STANDARD TABLE OF mara,
      gv_one   TYPE DECFLOAT16 LENGTH 8   DECIMALS 2,
      gv_second  TYPE p LENGTH 8  DECIMALS 0,
      gv_bool TYPE abap_bool.
ENDMETHOD.
```

will be cleaned to:

```
  METHOD b.
    DATA: gt_mat    TYPE STANDARD TABLE OF mara,
          gv_one    TYPE decfloat16             LENGTH 8 DECIMALS 2,
          gv_second TYPE p                      LENGTH 8 DECIMALS 0,
          gv_bool   TYPE abap_bool.
  ENDMETHOD.
```
I think the alignment of LENGTH with the end of the first line doesn't look good.

I hoped more for a cleaned version like the result of this (first line is not in the chain)

```
method a.
DATA gt_mat TYPE STANDARD TABLE OF mara.
DATA:      gv_one   TYPE DECFLOAT16 LENGTH 8   DECIMALS 2,
      gv_second  TYPE p LENGTH 8  DECIMALS 0,
      gv_bool TYPE abap_bool.     
ENDMETHOD.

```
Result:

```

  METHOD a.
    DATA gt_mat TYPE STANDARD TABLE OF mara.
    DATA: gv_one    TYPE decfloat16 LENGTH 8 DECIMALS 2,
          gv_second TYPE p          LENGTH 8 DECIMALS 0,
          gv_bool   TYPE abap_bool.
  ENDMETHOD.
```
Thanks!

**Comment by jmgrassau** [comment]
Hi, 

thanks a lot for this example, absolutely agree, the lengthy "TYPE STANDARD TABLE OF mara" should not influence the width of the entire TYPE column. Will look into this!

Kind regards,
Jörg-Michael

**Comment by jmgrassau** [comment]
Hi, 

thanks again for reporting this alignment issue – this should now be fixed with version 1.11.1, which was just released!

Kind regards,
Jörg-Michael
